jvm/sargon-android/src/main/java/com/radixdlt/sargon/extensions/BagOfBytes.kt

```diff
@@ -0,0 +1,25 @@
+@file:OptIn(ExperimentalUnsignedTypes::class, ExperimentalStdlibApi::class)
+
+package com.radixdlt.sargon.extensions
+
+import com.radixdlt.sargon.BagOfBytes
+import com.radixdlt.sargon.newBagOfBytesFrom
+import kotlin.random.Random
+
+fun bagOfBytesOf(byteArray: ByteArray) = newBagOfBytesFrom(bytes = byteArray)
+
+fun randomBagOfBytes(byteCount: Int) = Random.nextBytes(size = byteCount).toBagOfBytes()
```

Comment:
Is this CSRNG (**Cryptographically Secure** Random Number Generator)? I think not? Hmm maybe best put `randomBagOfBytes` being VisibilityForTesting (only) or otherwise DEBUG only compilation flag? What we need to protect against is using non-secure Randomness for Mnemonics. 

I dont think there is any API surface yet where you can pass a BagOfBytes for entropy which needs to be CSRNG but just want us to be mindful about that.

Reply:
Right will do that

Reply:
@CyonAlexRDX I could use [SecureRandom](https://developer.android.com/reference/kotlin/java/security/SecureRandom) provided by the android platform, but since this is only used in tests, I think it suffices to make it visible for tests only